docs/source/guide/clv/clv_intro.md

```diff
@@ -0,0 +1,48 @@
+2. Guide marketing campaigns.  Knowing what kinds of customers are most (and least) valuable provides a strong basis for selecting customers (e.g., segments) to target with future marketing campaigns. It also provides guidance about how much to spend when targeting specific types of customers.
+3. Guide retention and reacquisition efforts. Estimates of current customers' future value can be used to automate aspects of the company's customer relationship management system. High-value customers can be prioritized. Retention campaigns can target customers at risk of churning (terminating their relationship with the company).  Reacquisition campaigns can target high-value customers who have already churned.
+4. Avoid being misled by other factors. Without a broad, all encompassing way of valuing customers, companies may focus on KPIs that provide incomplete picture.  For example, companies may focus on the total number of customers acquired without realizing that many are low-value. Or companies may focus on the average revenue generated by a customer, without a good way of building forecasts of future behavior.
+5. Validate your understanding through predictions. We gain confidence in our knowledge of the world by making predictions and comparing them to what happens. CLV models generate forecasts that can be checked against reality. As a result, we can improve our understanding and modeling iteratively to become more accurate over time.
+
+## How does a Customer Lifetime Value model work?
+Customer lifetime value (CLV) models come in two basic varieties: contractual and non-contractual.  When customer relationships are contractual (e.g., subscription-based), the CLV modeling tends to be a bit more straightforward because the company always knows the status of the customer relationship (i.e., the relationship is ongoing or has been terminated). When the relationship is non-contractual, the status of the relationship is always uncertain.
+
+In either case, CLV models incorporate two main components: a model of revenue and a model of the lifetime.
+
+### Modeling revenue
+
+The revenue model characterizes the rate at which individual customers "buy" (i.e., generate transactions) and the magnitude of those purchases.  Typically, we can think about of an individual customer making purchases at a relatively steady rate (e.g., once per month). Of course, the actual timing of purchases will vary somewhat and we will be more confident about the rates associated with customers who have purchased many times in the past. The same is true of the magnitude of purchases.  Our estimates of frequency and magnitude allow us to characterize how many purchases to expect from a customer in the future and how large those purchases will be.  Together, this allows us to quantify the revenue expected from a customer if the relationship continued forever.
```

Comment:
```suggestion
The revenue model characterizes the rate at which individual customers "buy" (i.e., generate transactions) and the magnitude of those purchases.  Typically, we can think about of an individual customer making purchases at a relatively steady rate (e.g., once per month). Of course, the actual timing of purchases will vary somewhat and we will be more confident about the rates associated with customers who have purchased many times in the past. The same is true of the magnitude of purchases.  Our estimates of frequency and magnitude allow us to characterize how many purchases to expect from a customer in the future and how large those purchases will be.  Together, this allows us to quantify the revenue expected from a customer if the relationship continued indeterminately.
```
I believe that CLV models always infer/predict for some finite amount of future time